Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/ImageOpenedExEventArgs.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    public delegate void ImageOpenedExEventHandler(object sender, ImageOpenedExEventArgs e);
+
+    public class ImageOpenedExEventArgs : EventArgs
```

Comment:
It is strange but I didn't. I thought it is not enforced for EventArgs derived class or something.